kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -214,21 +345,32 @@ private void touch(File file) throws IOException {
      */
     public static void premain(String agentArgs) {
         int index = agentArgs.indexOf(':');
+        String[] args = agentArgs.split(":");
         if (index == -1) {
             LOGGER.error("Unable to parse arguments {}", agentArgs);
             System.exit(1);
         } else {
-            File brokerReadyFile = new File(agentArgs.substring(0, index));
-            File sessionConnectedFile = new File(agentArgs.substring(index + 1));
+            File brokerReadyFile = new File(args[0]);
+            File sessionConnectedFile = new File(args[1]);
+            String sslKeyStorePath = args[2];
+            String certStorePass = args[3];
+            String sslTrustStorePath = args[4];
```

Comment:
It would probably make more sense to me to have a separate password parameter for keystore and truststore.